src/problem2.rb

```diff
@@ -0,0 +1,9 @@
+class Problem2
+  def identify_largest_numbers(array, no_elements_identify)
```

Comment:
The solution asked for the first 2 numbers, no generic implementation was required.